contents/handbook/strategy/roadmap.md

```diff
@@ -42,10 +31,10 @@ We see a ton of usecases, like integrating PostHog into an existing data warehou
 There will be work on three main fronts:
 - Building plugins ourselves
 - Giving our community the tools to create their own
-- Promote adoption of these plugins.
+- Promote adoption of these plugins
+
+# 3. Make it easy to deploy a scalable version of PostHog, free
 
-# 3. Service huge volumes
+When we started PostHog we went with Postgres as our backend. This was a great choice early on as it allowed virtually anyone to deploy PostHog on any platform. Over time, however, the limitations of using Postgres for analytics became obvious, and we spent a chunk of last year converting our cloud offering to use Clickhouse in the backend.
```

Comment:
We don't have "last updated at" on these pages, so "last year" can be confusing.